js/tree.js

```diff
@@ -423,9 +428,20 @@
 				}
 
 			}
+		},
+
+		// This refreshes the children of a folder. Please destroy and re-initilize control for "root level" refresh.
+		// The data of the refreshed folder is not updated. This control's architecture only allows updating of children.
+		// Also, this function implies that the end-user wants the folder open.
+		refreshFolder: function refreshFolder($el) {
+				var $treebranch = $el.closest('.tree-branch');
+				var $refreshedBranch;
+				$treebranch.find('.tree-branch-children').eq(0).empty();
+				$refreshedBranch = this._discloseFolder($el.find('.tree-branch-header'));
```

Comment:
Don't need to actually open the folder. Just need to refresh the content in the folder. so just empty and run populate?